PWGCF/DataModel/FemtoDerived.h

```diff
@@ -56,14 +56,16 @@ static constexpr std::string_view ParticleTypeName[5] = {"Tracks", "V0", "V0Chil
 using cutContainerType = uint32_t; //! Definition of the data type for the bit-wise container for the different selection criteria
 
 DECLARE_SOA_INDEX_COLUMN(FemtoDreamCollision, femtoDreamCollision);
-DECLARE_SOA_COLUMN(Pt, pt, float);                    //! p_T (GeV/c)
-DECLARE_SOA_COLUMN(Eta, eta, float);                  //! Eta
-DECLARE_SOA_COLUMN(Phi, phi, float);                  //! Phi
-DECLARE_SOA_COLUMN(PartType, partType, uint8_t);      //! Type of the particle, according to femtodreamparticle::ParticleType
-DECLARE_SOA_COLUMN(Cut, cut, cutContainerType);       //! Bit-wise container for the different selection criteria
-DECLARE_SOA_COLUMN(PIDCut, pidcut, cutContainerType); //! Bit-wise container for the different PID selection criteria \todo since bit-masking cannot be done yet with filters we use a second field for the PID
-DECLARE_SOA_COLUMN(TempFitVar, tempFitVar, float);    //! Observable for the template fitting (Track: DCA_xy, V0: CPA)
-DECLARE_SOA_COLUMN(Indices, indices, int[2]);         //! Field for the track indices to remove auto-correlations
+DECLARE_SOA_COLUMN(Pt, pt, float);                         //! p_T (GeV/c)
+DECLARE_SOA_COLUMN(Eta, eta, float);                       //! Eta
+DECLARE_SOA_COLUMN(Phi, phi, float);                       //! Phi
+DECLARE_SOA_COLUMN(PartType, partType, uint8_t);           //! Type of the particle, according to femtodreamparticle::ParticleType
+DECLARE_SOA_COLUMN(Cut, cut, cutContainerType);            //! Bit-wise container for the different selection criteria
+DECLARE_SOA_COLUMN(PIDCut, pidcut, cutContainerType);      //! Bit-wise container for the different PID selection criteria \todo since bit-masking cannot be done yet with filters we use a second field for the PID
+DECLARE_SOA_COLUMN(TempFitVar, tempFitVar, float);         //! Observable for the template fitting (Track: DCA_xy, V0: CPA)
+DECLARE_SOA_COLUMN(Indices, indices, int[2]);              //! Field for the track indices to remove auto-correlations
+DECLARE_SOA_COLUMN(MinvLambda, minvLambda, float);         //! Field for the track indices to remove auto-correlations  // TEMPORARY
+DECLARE_SOA_COLUMN(MinvAntiLambda, minvAntiLambda, float); //! Field for the track indices to remove auto-correlations  // TEMPORARY
```

Comment:
If they are indexes shouldn't they be integers?

Reply:
Sorry, the comment  in code is wrong;  the column must be float as it is invariant mass. These columns are for testing purposes only, thus I would prefer to ignore this right now and in next PR I fix it. I would like this to be merged asap.

Reply:
@lauraser please fix the comments, it is not much work and the CI only needs 45 minutes...

Reply:
@jgrosseo I had to travel to a conference and knew that today won't find time during the day because of the conference too, that is why anticipated the delay in merge. I am sorry. Won't happen again. It is fixed now.